src/main/kotlin/com/devfloor/untitled/article/application/ArticleRequest.kt

```diff
@@ -10,14 +10,13 @@ data class ArticleRequest(
     val content: String,
     val hashtags: List<String>
 ) {
-    fun toArticle(articleRequest: ArticleRequest, user: User): Article {
-        return Article(
+    val isOptionsNotEmpty get() = options.isNotEmpty()
+
+    fun toArticle(articleRequest: ArticleRequest, user: User): Article =
+        Article(
             title = articleRequest.title,
             anonymous = articleRequest.anonymous,
             content = articleRequest.content,
             author = user
         )
```

Comment:
이건 객체의 메서드인데 자기 자신을 인자로 받을 필요가 있을까?

```suggestion
    fun toArticle(author: User): Article =
        Article(
            title = this.title,
            anonymous = this.anonymous,
            content = this.content,
            author = author
        )
```

Reply:
이것도 바꾸는게 좋겠다. 이 부분도 준형님 코드라 먼저 확인 하신 후에 반영하도록 할게 !

Reply:
네! 반영해주세욥!